athena-opensearch/src/main/java/com/amazonaws/athena/connectors/opensearch/OpensearchUtils.java

```diff
@@ -0,0 +1,62 @@
+/*-
+ * #%L
+ * athena-opensearch
+ * %%
+ * Copyright (C) 2019 - 2023 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.opensearch;
+
+import com.amazonaws.athena.connector.lambda.data.FieldBuilder;
+import org.apache.arrow.vector.types.Types;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.FieldType;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class OpensearchUtils
```

Comment:
if you don't expect this class's functionality to increase, you can just rename it more specific to OpensearchNestedStructBuilder so that it's better tied to its purpose.